<!-- -->

<launch>
	<node name="nuogeng_node" pkg="nuogeng" type="nuogeng_node" output="screen">
		<param name="port_name" value="/dev/ttyUSB2"/>
	</node>
	
	<node name="satellite_node" pkg="nuogeng" type="satellite_node" output="screen">
		<param name="port_name" value="/dev/ttyUSB3"/>
	</node>
	
	<node name="record_data_node" pkg="nuogeng" type="record_data_node" output="screen">
		<param name="use_synchronizer" value="True"/>
		<param name="record_duration" value="1.0"/>  <!-- s -->
		<param name="satellite_file" value="$(find nuogeng)/data/satellite.txt"/>
		<param name="gps_file" value="$(find nuogeng)/data/gps.txt"/>
	</node>
</launch>
